### DISCUSSION TITLE
用 Office Excel 打开 .csv 文件，中文变成乱码

### DISCUSSION THREAD

**Discussion by czm995212** [hiroi-sora/Umi-OCR#482] [Q&A] [answered]
使用 命令.\Umi-OCR.exe --screenshot --output test.csv导出
![Screenshot 2024-04-21 105751](https://github.com/hiroi-sora/Umi-OCR/assets/75152609/1f92c61a-b546-438c-a082-c292c55591e7)
![Screenshot 2024-04-21 105838](https://github.com/hiroi-sora/Umi-OCR/assets/75152609/e92b963e-1b2b-4629-bb3a-5c1db5870b80)
软件能正确识别出文字,但是在导出到csv后中文变成乱码,
识别内容为纯数字和英文导出正常,如果有汉字则变成乱码
用批量OCR 导出csv正常

**Comment by hiroi-sora** [comment]
麻烦用文本编辑器（如记事本）打开这个csv文件，看看编码是什么？

![image](https://github.com/hiroi-sora/Umi-OCR/assets/56373419/238b804f-9d33-4105-8d94-89f3e7485311)

**Reply by czm995212** [reply]
UTF-8

**Comment by hiroi-sora** [comment, marked as answer]
OK，这是微软的锅……

有三种方式解决：
方法1. 记事本 → 另存为 → 编码改为 `ANSI` 或者 `GBxxxxx` 。
方法2. Excel不要直接打开csv，而是“导入”它，参考 https://blog.csdn.net/henanlion/article/details/125812218
方法3. 放弃愚蠢的office，用WPS打开。或者先用WPS打开、保存一遍，再用office excel打开。

![image](https://github.com/hiroi-sora/Umi-OCR/assets/56373419/7b6109cf-4ee0-446e-8ba3-f18a796b0b56)
